src/Card/CardHeader.js

```diff
@@ -37,32 +37,25 @@ export default function CardHeader(props, context) {
   const classes = context.styleManager.render(styleSheet);
   const className = classNames(classes.cardHeader, classNameProp);
 
-  if (avatar) {
-    return (
-      <CardContent className={className} {...other}>
-        <div className={classes.avatar}>
-          {avatar}
-        </div>
-        <div className={classes.content}>
-          <Text type="body2" gutterBottom>
-            {title}
-          </Text>
-          <Text type="body2" secondary className={classes.contentSecondary}>
-            {subheader}
-          </Text>
-        </div>
-      </CardContent>
-    );
-  }
+  // adjustments that depend on the presence of an avatar
+  const titleType = avatar ? 'body2' : 'headline';
+  const subheaderType = avatar ? 'body2' : 'body1';
 
   return (
     <CardContent className={className} {...other}>
-      <Text type="headline">
-        {title}
-      </Text>
-      <Text type="body1" secondary>
-        {subheader}
-      </Text>
+      {avatar &&
+        <div className={classes.avatar}>
+          {avatar}
+        </div>
+      }
+      <div className={classes.content}>
+        <Text type={titleType}>
```

Comment:
![cardheader](https://cloud.githubusercontent.com/assets/3155127/24929359/a8bc2f0e-1ed3-11e7-9e86-d72d565bf844.png)

@oliviertassinari good catch, I missed that.  However, on closer inspection, doesn't it seem to add too much of a gap when compared to [the examples](https://storage.googleapis.com/material-design/publish/material_v_11/assets/0Bzhp5Z4wHba3bG0tNkIyNDhJZEU/components_cards28.png) in the spec?